testutil/network/util.go

```diff
@@ -55,11 +57,11 @@ func startInProcess(cfg Config, val *Validator) error {
 	if val.RPCAddress != "" {
 		node, ok := val.tmNode.(local.NodeService)
 		if !ok {
-			panic("can't cast service.Service to NodeService")
+			return fmt.Errorf("can't cast %T to NodeService", val.tmNode)
```

Comment:
@alexanderbez was there a particular reason for panicking here instead of returning an error? ref: https://github.com/cosmos/cosmos-sdk/pull/10210/files
same question for the change below

Reply:
Nope! This should be an error 👍